### DISCUSSION TITLE
response is too slow

### DISCUSSION THREAD

**Discussion by FAIpang** [acheong08/ChatGPT#966] [Q&A]
V1 response is too slow, can i do sometiing? VPN?

**Comment by acheong08** [comment]
How slow is it? Can you send a video? 
A couple possible reasons:
1. You do not have a Plus account. They recently decreased the speed for free accounts on purpose
2. Cloudflare is restricted in your region. If so, try out `CHATGPT_BASE_URL=https://chatgpt-proxy.fly.dev/ python3 -m revChatGPT.V1`. If that is also slow, then it must be an issue with OpenAI.

**Reply by FAIpang** [reply]
Thank your reply！
![image](https://user-images.githubusercontent.com/73394302/222097030-fe3f7a0c-bdee-44ee-9cda-b8fadbe32cf4.png)
This is my test_demo, a simple question took more than 20s

**Reply by acheong08** [reply]
Looks like an internet problem. Cloudflare slow in China perhaps?

**Reply by acheong08** [reply]
Try using a VPN

**Reply by FAIpang** [reply]
I use v2ray to set up a global proxy， but response is still slow. Do I need to set a proxy in the request?

**Reply by acheong08** [reply]
> I use v2ray to set up a global proxy， but response is still slow. Do I need to set a proxy in the request?

Set it in `config.json`

Alternative: V3 (Official API) just dropped with amazing speed

**Reply by FAIpang** [reply]
Thank you！ 
I have a question, now chatgpt free version only allows one to use at the same time can only ask one question, do Plus users have this limitation.

**Reply by acheong08** [reply]
Plus users have the same limitations. The difference is that Plus users have less per hour rate limits

**Reply by FAIpang** [reply]
OK! Thank you for your help!

**Comment by FAIpang** [comment]
CHATGPT_BASE_URL=https://chatgpt-proxy.fly.dev/， I don‘t understand this url？ Proxy？

**Reply by acheong08** [reply]
It's an alternative reverse proxy
